style.css

```diff
@@ -0,0 +1,386 @@
+  margin-left: -24px;
+}
+
+.greetings {
+  justify-content: center;
+  font-size: 40px;
+  line-height: 52px;
+  font-weight: 700px;
+  margin-top: unset;
+  margin-bottom: unset;
+  color: #172b4d;
+  font-family: 'Inter', sans-serif;
+}
+
+.job {
+  color: #36b37e;
+}
+
+.description {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: #172b4d;
+  font-family: 'Inter', sans-serif;
+}
+
+.socials {
+  display: flex;
+  gap: 1rem;
+  justify-content: center;
+  list-style-type: none;
+  margin-top: 14px;
+  padding-bottom: 2px;
+}
+
+.button {
+  color: #172b4d;
+  justify-content: center;
+  display: flex;
+  padding-bottom: 10px;
+}
+
+.headline {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding: 46px;
+  text-align: center;
+  overflow-y: auto;
+}
+
+.section-container9 {
+  background-image: url("images/illustrator-mobile.svg");
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
+}
+
+.background-image {
+  width: 100%;
+  position: absolute;
+}
+
+.line-break::after {
+  content: '';
+  width: 48px;
+  height: 4px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 24px;
+  background: #36b37e;
+  border-radius: 24px;
+  display: flex;
+  justify-content: center;
+}
+
+.works {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+  padding-top: 40px;
+}
+
+.section-placeholder {
+  margin-top: 50px;
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 58px;
+  margin-right: 24px;
+  margin-left: 24px;
+  align-items: center;
+  padding: 64px 24px 64px 24px;
+}
+
+.placeholder-container {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  padding: 10px;
+}
+
+.text-container {
+  border: 1px solid #ebf0ee;
+  width: 100%;
+  justify-content: center;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 12px;
+  margin-top: -10px;
+  background: white;
+}
+
+.header2 {
+  text-align: center;
+  color: #3a4a42;
+  padding-top: 15px;
+  font-weight: 600;
+  font-size: 24px;
+}
+
+.text {
+  align-items: center;
+  display: flex;
+}
+
+.button-text {
+  outline: none;
+  border: none;
+  background: #36b37f;
+  color: #ebf0ee;
+  border-radius: 4px;
+  width: 122px;
+  height: 48px;
+  font-size: medium;
+  margin-bottom: 36px;
+  transition-property: all;
+}
+
+.plane {
+  display: flex;
+  top: -60px;
+  position: relative;
+}
+
+.plane-container {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.text-aline {
+  display: flex;
+  justify-content: flex-end;
+}
+
+h3 {
+  text-align: center;
+  color: #172b4d;
+  padding: 40px;
+  font-size: 40px;
+}
+
+.h6 {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: #42526e;
+  padding-left: 24px;
+  padding-right: 24px;
+}
```

Comment:
- [ ] Just as a reminder the font for the texts in the "About me" section as I mentioned previously is not using the one given in the Figma design which is "Inter" so please make sure all the texts in that section are using the proper font. 😉